docs/github.md

```diff
@@ -0,0 +1,68 @@
+For multiple of these actions, you need to get the column_id for parameters. Here are the instructions on how to retrieve them.
+
+## Create personal access token
+
+Create a personal access token (classic) with permissions for:
+- repo
+- workflow
```

Comment:
This should not be needed. The token will not push YML files in `.github/workflow`
```suggestion
```